imagick_class.c

```diff
@@ -2424,7 +2424,7 @@ PHP_METHOD(Imagick, importImagePixels)
 		zend_throw_exception_ex(
 			php_imagick_exception_class_entry,
 				0,
-				"The map contains incorrect number of elements. Expected %ld, array has %d",
+				"The map contains incorrect number of elements. Expected %ld, array has %u",
```

Comment:
This is not actually correct for PHP 5.6 but better to be wrong just there and not wrong everywhere else... :)